src/js/actions/index.js

```diff
@@ -77,6 +79,26 @@ export function fetchReasonCodes() {
   };
 }
 
+export function fetchCurrencies() {
+  const url = '/openboxes/api/unitOfMeasure/currencies';
+  const request = apiClient.get(url);
+
+  return {
+    type: FETCH_CURRENCIES,
+    payload: request,
+  };
+}
+
+export function fetchOrganizations() {
+  const url = '/openboxes/api/organizations';
```

Comment:
@DominikaHatala, @jmiranda I think this should be debounced and we could return only org name and id. Currently, this request takes 2.5s and response has size of 1.6 MB (locally)